lib/shoryuken/default_worker_registry.rb

```diff
@@ -1,7 +1,7 @@
 module Shoryuken
   class DefaultWorkerRegistry < WorkerRegistry
     def initialize
-      @workers = {}
+      @workers = Concurrent::Hash.new
```

Comment:
I changed this because I'm worried https://github.com/allcentury/shoryuken/blob/712b35c4dd253ef275d7756760b604cd184d45bd/lib/shoryuken/extensions/active_job_adapter.rb#L34 is not thread safe.